module-4/backend/week_three.md

```diff
@@ -1,19 +1,32 @@
 ## Week Three - Module 4 Recap
-
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
-
-Note: When you're done, submit a PR. 
-
 1. At a high level, what is Node?
+  * Node is a runtime that allows for JavaScript to be run outside of a browser.
 2. What is Express? What is Express similar to in the Ruby world?
+  * Express is a framework and is similar to Rails in the Ruby world.
```

Comment:
And probably even more similar to Sinatra because it is lightweight and less opinionated than Rails!